client/components/recycleAndReuseComponents/Steps/StepThree/Geolocation/utils.js

```diff
@@ -97,44 +116,58 @@ export const getNearestNonBlueBinFacilities = (items, { lat, lng }) => {
 	const results = [];
 
 	for (const item of items) {
-		const validLocations = [];
+		let distance = null;
+		let nearestLocation = null;
 
 		for (const place of physicalChannels) {
 			if (
 				place.categories_accepted.includes(item.category) &&
 				place.type.includes(item.condition)
 			) {
-				validLocations.push({
-					id: place.id,
-					postal: place.postcode,
-					distance:
-						Math.round(
-							calcCrow(
-								lat,
-								lng,
-								place.latitude,
-								place.longitude,
-							) * 100,
-						) / 100,
-					latitude: place.latitude,
-					longitude: place.longitude,
-					address: place.address,
-					channel_name: place.channel_name,
-					operating_hours: place.operating_hours,
-					contact: place.contact,
-					website: place.website,
-					categories_accepted: place.categories_accepted,
-					organisation_name: place.organisation_name,
-					type: place.type,
-				});
+				let calcDist = Math.round(
+					calcCrow(
+						lat,
+						lng,
+						place.latitude,
+						place.longitude,
+					) * 100,
```

Comment:
only just started thinking about this *100 /100 part. is it some distance conversion? if so, can be added to part of the util logic instead of here

Reply:
It converts to coordinates to distance. You mean to move the logic to the utils.js file in client/components?